tasks.py

```diff
@@ -733,3 +738,30 @@ def link_checker(ctx: Context, skip_external: bool = True):
         skip_external=skip_external,
     )
     raise invoke.Exit(message, code)
+
+
+@invoke.task(
+    aliases=("automerge",),
+)
+def show_automerges(ctx: Context):
+    """Show github pull requests currently in automerge state."""
+    url = "https://api.github.com/repos/great-expectations/great_expectations/pulls"
+    response = requests.get(
+        url,
+        params={
+            "state": "open",
+            "sort": "updated",
+            "direction": "desc",
+            "per_page": 50,
+        },
+    )
```

Comment:
There's a good change users will forget the set their `GITHUB_TOKEN` env var - we could `raise_for_status` or honestly just keep it as is

Reply:
No need to overengineer

Reply:
Oh, yeah let me add the raise for status.